js/bibliographicCitationView.js

```diff
@@ -0,0 +1,674 @@
+                transform: rotate(180deg);
+            }
+            .referenceRemove {
+                color: red;
+            }
+        </style>
+        <div class="referenceDiv is-hidden">
+            <div class="col">
+                <p class="col-12 col-12-md">Gloss References are displayed below. Click the caret to see the rest of the reference. Click the pencil icon to edit the reference. Click the trash icon to remove the reference.
+                </p>
+                <button id="openCitationModalButton" class="smaller"> New Citation </button>
+                <div class="selectedEntities col-12 col-12-md">
+                </div>
+            </div>
+        </div?>
+        <div id="bibliographicCitationModal" class="bib-citation-modal">
+            <div class="bib-citation-modal-content">
+                <span class="bib-citation-close">&times;</span>
+                <form id="bibliographicCitationForm" deer-type="BibliographicResource" deer-context="http://purl.org/dc/terms">
+                    <div class="row">
+                        <label for="bibliographicCitationEditor" class="col-12 text-left">Bibliographic Citation: &#10082;</label>
+                    </div>
+            
+                    <div class="bibliographicCitation-container">
+                        <div class="row" id="bibliographicCitationtoolbar">
+                            <button type="button" class="toolbar-item" id="boldBtn" data-command="bold">Bold</button>
+                            <button type="button" class="toolbar-item" id="italicBtn" data-command="italic">Italics</button>
+                            <button type="button" class="toolbar-item" id="underlineBtn" data-command="underline">Underline</button>
+                            <button type="button" class="toolbar-item" id="linkBtn">Link</button>
+                            <button type="button" class="toolbar-item" id="subscriptBtn" data-command="subscript">Subscript</button>
+                            <button type="button" class="toolbar-item" id="superscriptBtn" data-command="superscript">Superscript</button>
+                        </div>
+                    
+                        <div class="row">
+                            <div id="bibliographicCitationEditor" class="col-12 required" contenteditable="true"></div>
+                        </div>
+                    </div>
+            
+                    <div class="row">
+                        <input type="hidden" deer-key="references" id="documentReference">
+                        <input id="citationSubmitButton" class="button primary" type="submit" value="Add Citation"/>
+                    </div>
+                </form>
+            </div>
+        </div>
+    `;
+  constructor() {
+    super();
+    this.citationsMap = {};
+  }
+  connectedCallback() {
+    const $this = this;
+    this.innerHTML = this.template;
+
+    var span = document.getElementsByClassName("bib-citation-close")[0];
```

Comment:
Instead of querying against the `document` you can scope to yourself -- that is you can query against `$this or this` which is safer and more reliable.

no `var`.  Do `let` or `const`

⚰️